app/routes/_layout+/bookings.new.tsx

```diff
@@ -131,6 +140,8 @@ export async function action({ context, request }: ActionFunctionArgs) {
       zone: hints.timeZone,
     }).toJSDate();
 
+    const assetIds = z.array(z.string()).parse(formData.getAll("assetIds"));
```

Comment:
This is not the correct way to do it. Please look at the example i gave you with bulk actions as well as how we usually parse data. The 'from' and 'to' you see above are exceptions, not the way to do it. you need to use `parseData` and add the `assetIds` to the `NewBookingFormSchema`.  
Please always check our other actions and how we do things so we dont have to do so many reviews for small things.

Reply:
Sure thing, have got the assetIds using the parseData and added the assetIds to the NewBookingFormSchema